code/game/turfs/simulated/floor/plating.dm

```diff
@@ -55,6 +55,11 @@
 				return
 	else if(istype(C, /obj/item/stack/tile))
 		if(!broken && !burnt)
+			for(var/obj/O in src)
+				if(O.level == 1) //ex. pipes laid underneath a tile
```

Comment:
is there a define for this `level` ?

Reply:
not that I have seen across pipe code where it's used, at least